.github/workflows/benchmarks.yml

```
@@ -16,8 +26,8 @@ jobs:
 
       uses: ./.github/workflows/interface-unit-tests.yml
       with:
-        # TODO: Update this in the future to a stable release or master branch
-        branch: add-pytest-benchmarks
+        # If running by workflow call, will check by provided ref_branch, if not will default to 'add-pytest-benchmarks'.
+        branch: ${{ inputs.ref_branch == '' && 'add-pytest-benchmarks' ||  inputs.ref_branch}}

```

```suggestion
        branch: ${{ inputs.ref_branch || 'add-pytest-benchmarks' }}
```

Due to how JS and expression evaluation works, if `inputs.ref_branch` is empty, it will fallback to the string.